test/basics/test.html

```diff
@@ -8,18 +8,24 @@
 		window.QUnit = window.parent.QUnit;
 		window.removeMyself = window.parent.removeMyself;
 	</script>
-	<script src="../../node_modules/steal/steal.js" main="test/basics/index.stache!done-autorender">
-		var AppViewModel = require("test/basics/state");
-		var canViewModel = require("can-view-model");
+	<script src="../../node_modules/steal/steal.js">
+		var Zone = require("can-zone");
 
-		var vm = canViewModel(document.documentElement);
+		new Zone().run(function() {
+			require('test/basics/index.stache!done-autorender');
```

Comment:
If your intention here is for the module to be executed inside of this new Zone you should use `System.import("foo")` instead.